app/assets/stylesheets/homepage.css.scss

```diff
@@ -43,6 +43,8 @@
     height: 550px;
     background: transparent image-url("homepage.png") no-repeat center -50px;
     background-size: cover;
+  box-shadow: 0px 30px 40px rgba(0,0,0,.5);
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>Commas in function arguments should be followed by a single space<br>`0px` should be written without units as `0`